Dev model

GRCaaaa opened the pull request:
## 新增内容
   锂离子固态电解质材料相关项目
   - 添加了完整的 XRDMatch 示例，用于 XRD 数据的半监督学习
   - 实现了 FlexMatch 算法，支持材料分类
   - 添加了 VGG 网络架构支持
   - 包含完整的代码、配置文件和文档

GRCaaaa committed:
Add XRDMatch example: XRD semi-supervised learning with FlexMatch

- Add complete XRDMatch implementation with FlexMatch algorithm
- Add VGG architecture support for XRD data processing
- Add comprehensive documentation and examples
- Support semi-supervised learning for Li-material classification

GRCaaaa committed:
Merge branch 'upstream_develop' into dev_model

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/PaddlePaddle/PaddleScience?pullRequest=1203) <br/>All committers have signed the CLA.

HydrogenSulfate commented:
@GRCaaaa 你好，感谢你提交PR，我这里发现这个PR好像文件的数量有点问题，你那里是正常的吗？
<img width="1192" height="346" alt="image" src="https://github.com/user-attachments/assets/2934765d-928c-43d5-8206-41eb5d861199" />

GRCaaaa committed:
Remove large files for successful push

- Remove saved model files directory (exp_1 to exp_9)
- Remove large model parameter files (.pdparams)
- Keep only code, configs, and documentation
- Files will be generated during training

HydrogenSulfate reviewed (commented):
你好，请使用pre-commit格式化你的代码

<img width="999" height="399" alt="Image" src="https://github.com/user-attachments/assets/38c08d04-7f23-47ed-8ab6-343205bbffc2" />

GRCaaaa committed:
Update XRD match implementation with complete PPSci framework integration

GRCaaaa committed:
Merge branch 'PaddlePaddle:develop' into dev_model

GRCaaaa committed:
Merge remote changes

GRCaaaa committed:
WIP: local changes before rebase

GRCaaaa committed:
修复代码格式问题及相关修改

leeleolay commented:
代码辛苦检查下，有太多额外的文件了

GRCaaaa committed:
更新xrdmatch模块，移除旧模型文件

HydrogenSulfate commented on competition/IJCAI_2024_CAR:
无关的内容请还原

HydrogenSulfate commented on jointContribution/DU_CNN/requirements.txt:
jointcontribution是无关内容，请还原

HydrogenSulfate commented on ppsci/externals/Open3D:
无关内容请还原

HydrogenSulfate commented on examples/ai4materials/xrdmatch/loss/ce_loss.py:
```diff
@@ -0,0 +1,15 @@
+        )
+
+    def forward(self, logits, targets):
+        return self.loss_fn(logits, targets)
```
这看起来并不是ppsci的代码

HydrogenSulfate reviewed (commented):
无关的内容请还原

GRCaaaa committed:
Merge remote-tracking branch 'upstream/develop' into dev_model

GRCaaaa committed:
remove ce_loss not used

leeleolay commented:
@GRCaaaa  辛苦再检查下代码

leeleolay commented on competition/IJCAI_2024_CAR:
请还原

leeleolay commented on examples/xrdmatch/configs/xrdmatch.yaml:
exampales下直接使用案例名，去掉子目录【ai4materials】

leeleolay commented on jointContribution/DU_CNN/requirements.txt:
无关文档请还原

leeleolay commented on ppsci/externals/Open3D:
无关请还原

leeleolay reviewed (changes requested):
需要改动的地方较多，辛苦仔细检查下，包括数据集，预训练模型权重文件是否需要上传到网上，辛苦提供原始文件的下载链接，我上传到云盘上提供链接可粘贴到文档中

GRCaaaa committed:
更改，数据和模型链接为通过网盘分享的文件：saved_models_ppsci等2个文件
链接: https://pan.baidu.com/s/1ot-7XQo304GZevBiXKT4hg?pwd=yvbk 提取码: yvbk
--来自百度网盘超级会员v5的分享

GRCaaaa committed:
Merge branch 'develop' into dev_model

GRCaaaa committed:
Merge branch 'dev_model' of https://github.com/GRCaaaa/PaddleScience into dev_model

leeleolay commented:
是否已经修改完成？我看提交的pr里面文件还是有多余的

GRCaaaa committed:
修改

GRCaaaa committed:
Merge branch 'develop' into dev_model

GRCaaaa committed:
fix: remove unrelated directories from PR

GRCaaaa committed:
Merge branch 'dev_model' of https://github.com/GRCaaaa/PaddleScience into dev_model

GRCaaaa committed:
“恢复之前误删的内容 --no-verify

git commit -m  “恢复之前误删的内容

GRCaaaa committed:
提交本次内容

GRCaaaa committed:
提交本次内容

GRCaaaa committed:
Merge branch 'PaddlePaddle:develop' into dev_model

GRCaaaa committed:
修改

GRCaaaa committed:
Merge branch 'dev_model' of https://github.com/GRCaaaa/PaddleScience into dev_model
